scripts/publish/src/publishConfig.js

```diff
@@ -0,0 +1,17 @@
+module.exports = packageJson => {
+  let access = 'public';
+  let tag = 'latest';
+
+  // Set access and tag based on value in package.json config.
+  if (packageJson.publishConfig) {
+    if (packageJson.publishConfig.access) {
+      ({ access } = packageJson.publishConfig);
+    }
```

Comment:
As a future improvement it would be nice to have an else like so:
```
else {
    console.log("No access value found in config. Using default access value of 'public'");
}
```